core/src/main/java/com/linecorp/armeria/common/util/Version.java

```diff
@@ -61,7 +61,7 @@
  * it may not be possible to retrieve the information completely, depending on your environment, such as
  * the specified {@link ClassLoader}, the current {@link SecurityManager}.
  */
-public final class Version {
+public final class Version implements Comparable<Version>{
```

Comment:
I will add a space here after the `<Version>`. Will set up the formatter in the IDE.